### PR TITLE
8104 normalization exception

### PR DESCRIPTION
**Pull request by gdicristofaro**

### CODE REVIEW

**Comment by eugene7646** [comment]
@gdicristofaro There is a merge conflict now.

**Comment by gdicristofaro** [comment]
> @gdicristofaro There is a merge conflict now.

Ok.  I think this should be resolved now.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java [hunk 1]
```diff
@@ -364,7 +364,7 @@ private static boolean isSystemOsAccount(String accountAddr) {
                     correlationAttrs.addAll(makeCorrAttrsFromCommunicationArtifact(artifact, attributes));
                 }
             } catch (CorrelationAttributeNormalizationException ex) {
-                logger.log(Level.SEVERE, String.format("Error normalizing correlation attribute (%s)", artifact), ex); // NON-NLS
+                logger.log(Level.WARNING, String.format("Error normalizing correlation attribute (%s): %s", artifact, ex.getMessage())); // NON-NLS
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java:367]: For consistency, I think the InvalidAccountIDException below this should also be a warning.

Comment (gdicristofaro) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java:367]: Sounds good.

**Pull request merged**